runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -344,6 +344,12 @@ public void onChatMessage(ChatMessage event)
 			String fileName = "Kill " + format(new Date());
 			takeScreenshot(fileName);
 		}
+
+		if (config.screenshotBossKills() && chatMessage.contains("kill count is:"))
```

Comment:
This will not capture all activities with kill counts. [See the chat commands plugin.](https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java#L85-L88) Moving those to a common location for shared use among plugins would probably be a good way to approach that.

Reply:
We do imo not want to screenshot raids kc since we are already taking screenshots when the loot is received, which i assume is the reason people would want to screenshot boss kc.

Reply:
And screenshotting e.g wintertodt kc is useless

Reply:
That's fair, though we should still be [reusing the boss kill count regex](https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java#L85-L88).